utility.py

```diff
@@ -0,0 +1,75 @@
+from concurrent import futures
```

Comment:
missing unit tests

Reply:
we are trying to create folder for user account. will do unit tests after this